cmd/request/request.go

```diff
@@ -14,17 +18,21 @@ const (
 
   Send a custom API request to the configured PingOne tenant, making a GET request to retrieve JSON configuration for a specific environment.
     pingcli request --service pingone --http-method GET --output-format json environments/$MY_ENVIRONMENT_ID
+	
+	Send a custom API request to the configured PingOne tenant, making a POST request to create a new environment with JSON data sourced from a file.
+		pingcli request --service pingone --http-method POST --data @./my-environment.json environments
```

Comment:
`@` symbol should not be used in logic code or documented with the addition of --data-raw flag